### DISCUSSION TITLE
Enhancement on upload collections sorting experience in admin panel

### DISCUSSION THREAD

**Discussion by cmngan** [payloadcms/payload#1749] [Feature Requests & Ideas]
Currently, in admin panel, upload collections have different UI from others. It is common that users want to sort it by last updated time. Although it is feasible to add a sort condition from the ui, it is not very user friendly. i.e. There are at least 3 clicks shown in the pic below: 
<img width="875" alt="image" src="https://user-images.githubusercontent.com/20035088/209171102-4ba7654e-129a-4809-b53e-35ec1f04c066.png">

Here are some possible suggestions:
1. Allow settings default sorting field in collection config
2. Save the previous sorting conditions so that users only need to set once
3. Redesign the upload ui so that fewer clicks can achieve that. e.g. In other collections, since it is table view, users can click on the table head to add sorting.

**Comment by DanRibbens** [comment]
We are making changes with our bulk operations work that's coming soon that replaces the upload list view. Uploads going forward will be listed in the same table list as other collections with an added thumbnail cell.

I believe this will solve the UX issue here.
